src/shared/style/_common.scss

```diff
@@ -2,7 +2,7 @@
 
 $co-external-link-padding-right: 15px;
```

Comment:
@vikram-raj do you know if we are using this var anywhere? I wasn't able to find any.